CMakeLists.txt

```diff
@@ -51,6 +51,9 @@ add_cts_option(SYCL_CTS_ENABLE_EXT_ONEAPI_TESTS
 add_cts_option(SYCL_CTS_ENABLE_KHR_TESTS
     "Enable all extension Khronos tests" OFF)
 
+add_cts_option(SYCL_CTS_SYCL_NEXT_TESTS
+    "Enable all SYCL Next tests" OFF)
```

Comment:
I wonder if it would better to make it a SYCL version option, i.e. an option that would let the user switch between SYCL 2020, SYCL Next, SYCL Next Next, SYCL Next Next Next, ...

Reply:
WG discussed and converged on keeping this name.